plugin/tiff/src/main/java/it/geosolutions/imageioimpl/plugins/tiff/TIFFImageReader.java

```diff
@@ -73,6 +73,8 @@
  */
 package it.geosolutions.imageioimpl.plugins.tiff;
 
+import com.sun.media.imageioimpl.common.ImageUtil;
+import com.sun.media.imageioimpl.common.PackageUtil;
```

Comment:
Are these imports really needed?

Reply:
Yes, they are needed down. IntelliJ "optimized" the imports in a different way than Eclipse, that's why you get them here... but they were already there.

Reply:
right sorry. Didn't notice the -import counterpart there below.